src/k8s-dashboard/deploy/k8s-dashboard.yaml.template

```diff
@@ -38,11 +38,14 @@ spec:
       containers:
       - name: kubernetes-dashboard-openpai
         image: k8s.gcr.io/kubernetes-dashboard-amd64:v1.10.1
-        command:
-          - /dashboard
-          - --insecure-bind-address=0.0.0.0
         args:
-          - --insecure-port=9090
+          - --auto-generate-certificates
+          - --port={{ cluster_cfg['k8s-dashboard']['dashboard-port'] }}
+          - --tls-cert-file=/tls.crt
```

Comment:
If set tls cert path as the value ```/tls.crt```, dashboard will find it in ```/certs/tls.crt```? I'm very confused about this, can you remove the option ```--auto-generate-certificates``` and try again. I suspect the crt and key is generated by k8s-dashboard
'